apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -103,13 +99,23 @@ const modalStoryMeta: ModalStoryMeta = {
             description: 'Content to place within the modal',
             control: 'text',
         },
-        leadingAction: {
-            description: 'The leading action configuration for the modal.',
-            control: 'object',
+        leadingActionText: {
+            description: 'The text to display on the leading action button.',
+            control: 'text',
         },
-        supportingAction: {
-            description: 'The supporting action configuration for the modal. Will not appear if no leading action is provided.',
-            control: 'object',
+        leadingActionVariant: {
+            description: 'The variant of the leading action button.',
+            control: 'select',
+            options: buttonVariants,
+        },
+        supportingActionText: {
+            description: 'The text to display on the supporting action button.',
+            control: 'text',
+        },
+        supportingActionVariant: {
+            description: 'The variant of the supporting action button.',
+            control: 'select',
+            options: buttonVariants,
         },
         aria: {
             description: 'The ARIA labels used for the modal close and back buttons, as well as loading state.',
```

Comment:
Will need to update this description, as it also includes the leading/supporting action labels too.

Reply:
Ah yep 👍🏻 I updated this elsewhere but not here